voice/voice-ai/x/AppSettings/SettingsView.swift

```diff
@@ -91,30 +106,31 @@ struct SettingsView: View {
         }
         .padding()
     }
-    
+
     private func purchaseOptionsPopoverContent() -> some View {
         VStack {
             Text("settingsView.purchaseMenu.title").font(.headline)
             Divider()
             Button("settingsView.purchaseMenu.pay5viaApple") { showPurchaseDialog() }
             //                Button("Restore Purchase") { /* Add logic for restoring purchase */ }
             Button("settingsView.purchaseMenu.signInAccount") { performSignIn() }
-            Button("settingsView.purchaseMenu.deleteAccount") { 
+            Button("settingsView.purchaseMenu.deleteAccount") {
                 appSettings.isPopoverPresented = false
-                showDeleteAccountAlert()                     }
+                showDeleteAccountAlert()
+            }
             Button("button.cancel", role: .cancel) {
                 appSettings.isPopoverPresented = false
             }
         }
         .padding()
     }
-    
+
     func actionSheet() -> ActionSheet {
         return ActionSheet(title: Text("settingsView.mainMenu.title"), buttons: [
             .cancel({
                 appSettings.showSettings(isOpened: false)
             }),
-            .default(Text("settingsView.mainMenu.shareTranscript")) { saveTranscript() },
+            .default(Text("settingsView.mainMenu.shareTranscript")) { shareLogs() },
```

Comment:
Is there another place where transcript is exported? The label still says transcript